packages/@aws-cdk/aws-eks/lib/helm-chart.ts

```diff
@@ -51,6 +51,7 @@ export interface HelmChartOptions {
 
   /**
    * The values to be used by the chart.
+   * For nested values use a nested dictionary, i.e. { installationCRDs: true, webhook: { port: 9443 } }
```

Comment:
This looks redundant here and lacks the formatting available via README (the values' format is expressed in code 3 lines down).

Reply:
@Naumel I can rewrite it to match the formatting I used in the README. As for the redundancy, it would have been helpful for me as a Typescript novice since the typing provided in line 57 doesn't _necessarily_ exclude using dot notation.